src/service/viewer-impl.js

```diff
@@ -766,16 +759,15 @@ export class Viewer {
     if (eventType == 'viewport') {
       if (data['paddingTop'] !== undefined) {
         this.paddingTop_ = data['paddingTop'];
```

Comment:
Do we even want `initPaddingTop` value here? Why not just read this value in viewport class using viewer.getParam()? It doesn't look like viewer ever cares about padding.